### PR TITLE
[ECO-1866] Assorted docs

### PR DESCRIPTION
**Pull request by kristinatub**
# Description

Add initial docs set

# Testing

Local build, CI

### CODE REVIEW

File: doc/doc-site/docs/resources/faq.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+A:  45,000 APT Market Cap
+
+### Q: Assuming nobody sells their tokens, how much buy pressure is required for the Emojicoin reach a 45,000 APT market cap?
+A:  10,000 APT
+
+### Q: If an Emojicoin market is already launched, can I launch a second one of the same Emoji?
+A:  No, once a particular Emojicoin trading pair is launched, you cannot launch another. You should potentially explore launching Emojicoin combinations instead. Get creative, don't be lame. 
+
+### Q: Are emojicoins the future? 
+A: Emojis are a universal language, transcending borders and languages. 🌍 With the rise of decentralized finance and the integration of emojicoins, I believe will see a global shift where emojicoins become the standard for transactions. So get ready to live, laugh, and love with emojicoins! They're not just a trend; they're a revolution. Embrace the future, where every transaction is a smile, a laugh, and a story. Emojicoins are here to stay, and they're going to change the world one emoji at a time. 🌈🚀💖
```
Comment (xbtmatt) [inline doc/doc-site/docs/resources/faq.md:23]:
```suggestion
A: Emojis are a universal language, transcending borders and languages. 🌍 With the rise of decentralized finance and the integration of emojicoins, we believe we could witness a global shift where emojicoins become the standard for transactions. So get ready to live, laugh, and love with emojicoins! They're not just a trend; they're a revolution. Embrace the future, where every transaction is a smile, a laugh, and a story. Emojicoins are here to stay, and they're going to change the world one emoji at a time. 🌈🚀💖
```

Should be third person here, right?

File: doc/doc-site/docs/resources/glossary.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+slug: /glossary
+title: 🍆 Glossary
+hide_title: False
+description: Documentation for emojicoin dot fun
+---
+Degenerate vocabulary you must know
```
Comment (xbtmatt) [inline doc/doc-site/docs/resources/glossary.md:7]:
```suggestion
Degenerate jargon you must know
```

I think jargon is a little more fitting here

Maybe even something like:

~~Degen~~ Defi jargon you must know

File: doc/doc-site/docs/resources/glossary.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+**Bonding Curve**: smart contracts that determine a token's price based on its circulating supply. In this case the first state of the emojicoin will be here before transitioning into the CPAMM.
+
+**Concentrated Liquidity Automated Market Maker (CLAMM)**:More efficient AMM design where users have more control over price where liquidity is placed.
+
+**Constant Product Automated Market Maker (CPAMM)**: Most popular AMM desgin univ2 style.
```
Comment (xbtmatt) [inline doc/doc-site/docs/resources/glossary.md:13]:
```suggestion
**Constant Product Automated Market Maker (CPAMM)**: The most popular AMM design- Uniswap v2 style.
```

File: doc/doc-site/docs/resources/glossary.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+Where, ‘x’ and ‘y’ represent the quantities of two assets in a liquidity pool, and ‘k’ is a constant. 
+
+
+**Degen**: Someone who LARPS as a gambler but doesn't actually make any money.
```
Comment (xbtmatt) [inline doc/doc-site/docs/resources/glossary.md:17]:
```suggestion
**Degen**: pronunciation: DEE-jen (/ˈdiː.dʒɛn/). Someone who LARPS as a gambler but doesn't actually make any money. 
```

Not necessary of course- just an idea. Also this is funny 😆

File: doc/doc-site/docs/start-here/bridging-from-solana.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+---
+slug: /bridging-from-solana
+title: 🌈 Bridging From Solana to Aptos
+hide_title: False
+description: Documentation for emojicoin dot fun
+---
+
+Stay tuned for future LayerZero Solana  Aptos Integration announcements.
```
Comment (xbtmatt) [inline doc/doc-site/docs/start-here/bridging-from-solana.md:8]:
```suggestion
Stay tuned for future LayerZero Solana ↔️ Aptos Integration announcements.
```

File: doc/doc-site/docs/start-here/bridging-from-solana.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+
+In the meantime see the resources below
```
Comment (xbtmatt) [inline doc/doc-site/docs/start-here/bridging-from-solana.md:10]:
```suggestion
In the meantime see the resources below.
```

**Review by xbtmatt — COMMENTED** [review]
This is great! Tons of stuff here, I was curious to read through it so I left a few comments and suggestions as I was going through it 😄

**Review by alnoki — APPROVED** [review]

**Pull request merged**